<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PivoPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	color: white;
    background-color: #2b2b2b;
    border-width: 5px;
    border-color: #afafaf;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="MusicList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>201</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget
{
	color: white;
	background-color: #2b2b2b;
}
QScrollBar:vertical {              
    border: none;
    background:white;
    width:3px;
    margin: 0px 0px 0px 0px;
}
QScrollBar::handle:vertical {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
    stop: 0 rgb(5, 35, 0), stop: 0.5 rgb(21, 90, 0), stop:1 rgb(5, 35, 0));
    min-height: 0px;
}
QScrollBar::add-line:vertical {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
    stop: 0 rgbrgb(21, 90, 0), stop: 0.5 rgb(21, 90, 0),  stop:1 rgbrgb(21, 90, 0));
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
    background: rgb(5, 35, 0)
    height: 0 px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionRectVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="QueueList">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>201</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget
{
	color: white;
	background-color: #2b2b2b;
}
QScrollBar:vertical {              
    border: none;
    background:white;
    width:3px;
    margin: 0px 0px 0px 0px;
}
QScrollBar::handle:vertical {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
    stop: 0 rgb(5, 35, 0), stop: 0.5 rgb(21, 90, 0), stop:1 rgb(5, 35, 0));
    min-height: 0px;
}
QScrollBar::add-line:vertical {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
    stop: 0 rgbrgb(21, 90, 0), stop: 0.5 rgb(21, 90, 0),  stop:1 rgbrgb(21, 90, 0));
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
    background: rgb(5, 35, 0)
    height: 0 px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addMusicButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    background-color: #2b2b2b;
    border-width: 5px;
    border-color: #afafaf;
}</string>
    </property>
    <property name="text">
     <string>Add files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="repeatButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    background-color: #2b2b2b;
    border-width: 1px;
    border-color:  #1f1f1f;
	border-style: solid;
}</string>
    </property>
    <property name="text">
     <string>⥁</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    background-color: #2b2b2b;
    border-width: 1px;
    border-color:  #1f1f1f;
	border-style: solid;
}</string>
    </property>
    <property name="text">
     <string>«</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    background-color: #2b2b2b;
    border-width: 1px;
    border-color:  #1f1f1f;
	border-style: solid;
}</string>
    </property>
    <property name="text">
     <string>ll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    background-color: #2b2b2b;
    border-width: 1px;
    border-color:  #1f1f1f;
	border-style: solid;
}</string>
    </property>
    <property name="text">
     <string>»</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shuffleButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    background-color: #2b2b2b;
    border-width: 1px;
    border-color:  #1f1f1f;
	border-style: solid;
}</string>
    </property>
    <property name="text">
     <string>⤱</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {

	height: 20px;
    border-radius: 8px;
}

QSlider::groove:horizontal {
	height: 3px;
	background: #646464;
	border-radius: 13px;
	border: 1px solid #313131;
}

QSlider::handle:horizontal {
	border: 1px solid #212529; 
	height: 4px;
	width: 8px; 
	border-radius: 4px; 
	background: #fbfffa;  
	margin-top: -4px; 
	margin-bottom: -4px;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="TimingSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>26</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {

	height: 20px;
    border-radius: 8px;
}

QSlider::groove:horizontal {
	height: 3px;
	background: #646464;
	border-radius: 13px;
	border: 1px solid #313131;
}

QSlider::handle:horizontal {
	border: 1px solid #212529; 
	height: 4px;
	width: 8px; 
	border-radius: 4px; 
	background: #fbfffa;  
	margin-top: -4px; 
	margin-bottom: -4px;
}</string>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timing_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeleft_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="songname_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addPlaylistButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    background-color: #2b2b2b;
    border-width: 1px;
    border-color:  #1f1f1f;
	border-style: solid;
}</string>
    </property>
    <property name="text">
     <string>new playlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pickPlaylistButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    background-color: #2b2b2b;
    border-width: 1px;
    border-color:  #1f1f1f;
	border-style: solid;
}</string>
    </property>
    <property name="text">
     <string>pick playlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setDefaultButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
    background-color: #2b2b2b;
    border-width: 1px;
    border-color:  #1f1f1f;
	border-style: solid;
}</string>
    </property>
    <property name="text">
     <string>default</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border: none;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
